### ISSUE TITLE
No error handling directive

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#947]
Not sure why the following block is tripping the 'no error handling directive rule':
````
REPEAT
   ON STOP   UNDO, RETRY
   ON ERROR  UNDO, RETRY
:
   // false positive
END.
````
If the ``on stop`` is removed:
````
REPEAT
   ON ERROR  UNDO, RETRY
:
   // ok
END.
````
Or the ``on error`` is moved to before ``on stop``:
````
REPEAT
   ON ERROR  UNDO, RETRY
   ON STOP   UNDO, RETRY
:
   // ok
END.
````
Then the rule is happy.

**Comment by gquerret** [comment]
Reproduced ! It looks like a change in the syntax tree introduced this regression, the bad pattern was incorrectly detected by the rule.

**Comment by gquerret** [comment]
Fixed in develop
